paddle/framework/program_desc.cc

```diff
@@ -64,5 +67,27 @@ ProgramDesc::ProgramDesc(const std::string &binary_str) {
   }
 }
 
+const std::vector<std::string> ProgramDesc::GetFeedVarNames() {
+  BlockDesc *global_block = blocks_[0].get();
+  std::vector<std::string> feed_var_names;
+  for (auto *op : global_block->AllOps()) {
+    if (op->Type() == "feed") {
```

Comment:
feed -> kFeedOpType
and let's rename `feed_var_names` to `feed_target_names` if there is no better candidate.